.gitignore

```diff
@@ -3,3 +3,4 @@
 .DS_Store
 .sass-cache
 build/
+node_modules
```

Comment:
seems that even though you've added node modules to gitignore, they stil lgot push - could you take a look at that?
also please add .tmp to gitignore - some files form there also got pushed

Reply:
I added the tmp to gitignore just now, I made a mistake when I added the node_modules to gitignore, I pushed some of them before I added it to the gitignore file, can I remove them from the repo somehow?

Reply:
I removed the node_modules from the repo I think, please check when you have time